### ISSUE TITLE
사장님 마이페이지

### ISSUE THREAD

**Issue by Wo-oJin** [Wo-oJin/Ari#25]
필요한 기능

1. 가게 정보 추가(e.g. 이미지, 한 줄 정보, 가게 영업시간, 개인 이벤트(내용, 기간, 대상))
2. 가게 정보 수정(e.g. 가게 이름, 가게 주소, 사장님 성함, 사장님 전화번호)

**Comment by Wo-oJin** [comment]
멤버 : 가게의 관계가 1:1 -> 1:N으로 변경됨에 따라,
최초 로그인 시 1개의 가게 정보만 입력받고 나머지 가게는 마이페이지에서 입력받으면 될 것 같습니다.

ex) a, b, c 가게를 운영하시는 member1 사장님이 회원가입 하실 땐 a 가게 정보만 입력하고 마이페이지에서 '가게 추가' 버튼을 눌러 b, c 가게 정보를 추가로 등록하기

**Comment by ckd0325** [comment]
저도 그런 방향으로 생각을 해봤는데 저희는 가입당시에 기존에 설정한 인증코드를 사장님에게 전달하고 가입이 이뤄지잖아요? 그말은 우선 가입을 하고 난 뒤에는 정보 수정을 통해 자유롭게 주소나 가게 이름을 수정할 수 있다는 뜻이기도 합니다.

  이런 상황에서 만약에 가게 사장님들끼리 사이가 좋지 않다거나 누군가 악의로 사장님 계정에 로그인을 했을 때, 남의 가게와 똑같은 주소로 가게를 추가하고 이름이나 이벤트 내용 등을 극단적으로 바꾸어 버리면 원인 추적이 힘들 것 같습니다. 반면 한 계정이 하나의 가계만 가진다면 누군가 가게 주소나 정보를 바꾸어도 추적이 어렵지는 않을 것 같습니다.

 또 제휴 이벤트 관계를 맺었을 때 n개의 가게중 어떤 가게와 협약을 할 건지 선택하는 과정도 추가되기도 하고요. 일단 제 생각은 가게 이름이나 주소는 초기에 생성할 때만 받고 이후 수정을 불가능하게 하는 식으로 해야 할 것 같아요. 아님 두개 이상의 가게를 소유하는 경우는 드무니깐 한계정당 하나의 가계만 갖게 해도 될 것 같구요

**Comment by Wo-oJin** [comment]
@ckd0325 

네 충분히 발생할 수 있는 문제라 생각합니다. 근데 저번 회의 때 기획팀 분들이 말씀해주셨던 것처럼, 현재 아리단길 내엔 사장님 한 분이 여러 가게를 가지고 있는 경우가 많은 것 같습니다. 그래서 1차 테스트 전에 1:N 관계로 변경하는 게 좋을 것 같고, 말씀해주신 보안 이슈는 가게 추가를 할 때마다 각 가게마다 부여된 고유의 인증 코드를 입력하는 식으로 하면 될 것 같습니다.
